packages/react/src/use-chat.ts

```diff
@@ -513,12 +516,58 @@ By default, it's set to 0, which will disable the feature.
 
       event?.preventDefault?.();
 
+      let parts: ContentPart[] = [];
+
+      if (input) {
+        parts.push({
+          type: 'text',
+          text: input,
+        });
+      }
+
+      if (options.files) {
```

Comment:
My preference here would be to set/send parts only when files is != undefined. Otherwise we change what is being send for existing users, increases their traffic unexpectedly.

Reply:
done!